src/app/storymaps/tpl/builder/settings/ViewMapOptions.html

```diff
@@ -36,4 +36,23 @@
 			</a>
 		</label>
 	</div>
+      
+    <div class="" style="">
+      <label>
+        Maptiks Trackcode: <input type="text" class="form-control input-sm maptiks-trackcode" />
+        <a href="#" class="help" data-toggle="tooltip" data-placement="right" title="Enter Maptiks trackcode">
+            <img class="helpImg" src="resources/tpl/builder/icons/builder-help.png">
+        </a>
+      </label>
+	</div>
+      
+    <div class="" style="">
+      <label>
+        Maptiks ID:   
+        <input type="text" class="form-control input-sm maptiks-id" />
```

Comment:
lets change this to use the 'id' attribute rather than the 'class'. 

ie: <input id="maptiks-id">